tests/clv/models/test_beta_geo.py

```diff
@@ -83,14 +83,30 @@ def test_model(self, a_prior, b_prior, alpha_prior, r_prior):
             "r_log__": (),
         }
 
-    def test_numerically_stable_logp(self):
+    def test_customer_id_warning(self):
+        with pytest.raises(ValueError):
```

Comment:
Add a `match=`?
```suggestion
        with pytest.raises(ValueError, match=...):
```

Reply:
Added!

Reply:
I don't see it?

Reply:
My bad, I messed up a rebase. It should be there now